dl1_data_handler/image_mapper.py

```diff
@@ -91,6 +97,8 @@ def __init__(
         self.camera_type = self.geometry.name
         self.n_pixels = self.geometry.n_pixels
         # Rotate the pixel positions by the pixel to align
+        if self.camera_type == "UNKNOWN-7987PX":
```

Comment:
Can you maybe overwrite the name of the geometry after reading in the camera geometry object from the file, i.e. in the reader?